<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mfDosageDialog</class>
 <widget class="QDialog" name="mfDosageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>635</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="iconInteractionLabel">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="drugNameButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="dosageSelectorLayout">
     <item>
      <widget class="QLabel" name="dosageSelectorLabel">
       <property name="text">
        <string>Select a predefined dosage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dosageSelectorCombo"/>
     </item>
     <item>
      <widget class="QPushButton" name="but_ModifyDosage">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_CreateDosage">
       <property name="text">
        <string>Create new</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabIntakes">
      <attribute name="title">
       <string>Intakes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Intakes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="fromToIntakesCheck">
            <property name="text">
             <string>From x To y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="intakesFromLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>From</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="intakesFromSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="intakesToLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>To</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="intakesToSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="intakesCombo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="userformsButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="intakesFromLabel_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Each</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="periodSpin"/>
            </item>
            <item>
             <widget class="QComboBox" name="periodSchemeCombo">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Duration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="margin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="fromToDurationCheck">
            <property name="text">
             <string>From x To y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="durationFromLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>From</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="durationFromSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="durationToLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>To</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="durationToSpin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="durationCombo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExtras">
      <attribute name="title">
       <string>Extras</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Extra-datas</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" rowspan="4">
           <widget class="tkStringListView" name="dailySchemeListView" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="administrationCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="ALDCheck">
              <property name="text">
               <string>100%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="INNCheck">
              <property name="text">
               <string>INN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="noFormCheck">
              <property name="text">
               <string>No form</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="noPresentationCheck">
              <property name="text">
               <string>No presentation</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Meal time relation</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="mealTimeCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTextEdit" name="resultTextBrowser">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>322</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTextEdit" name="noteTxtEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="toogleInnButton">
     <property name="text">
      <string>Toogle to INN</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tkStringListView</class>
   <extends>QWidget</extends>
   <header>tkStringListView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mfDosageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mfDosageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromToIntakesCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>intakesFromLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromToIntakesCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>intakesToLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromToIntakesCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>intakesToSpin</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromToDurationCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>durationFromLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromToDurationCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>durationToLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromToDurationCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>durationToSpin</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
